packages/material-ui/src/MenuList/MenuList.test.js

```diff
@@ -72,7 +72,7 @@ describe('<MenuList />', () => {
       assert.strictEqual(list.style.paddingLeft, '');
       assert.strictEqual(list.style.width, '');
       menuListActionsRef.current.adjustStyleForScrollbar(
-        { clientHeight: 10 },
+        { clientHeight: 20 },
```

Comment:
/cc @ryancogswell 

Not entirely sure why. I think it has something to do with `ref` now being attached to the actual host component instead of the class wrapper. Without this change browser test were failing: https://circleci.com/gh/mui-org/material-ui/79900

Reply:
Prior to the change, no CSS was injected in the page, now, it does. I have already seen this behavior in the past. Because I was newly injecting CSS, the layout values were different but correct.

Reply:
Makes the change even better I suppose.

Reply:
Hum, but the MenuList has no styles. It's not related. I can't explain it.